packages/mobile/src/components/Header.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+    StyleSheet,
+    SafeAreaView,
+    View,
+    Text,
+    TouchableOpacity,
+} from 'react-native';
+import SistemaLogo from '../../assets/SistemaLogo';
+import HomeIcon from '../../assets/HomeIcon';
+import Home from '../home/Home';
+import { Dimensions } from 'react-native';
+import OptionIcon from '../../assets/OptionIcon';
+
+const height = Math.floor(Dimensions.get('window').height / 10);
+
+const Header = ({ navigation }) => {
+    return (
+        <SafeAreaView style={styles.container}>
+
+            {/* touchableopacity wrapper for setting icon */}
+            <TouchableOpacity style={styles.settingContainer}>
+
+                <View style={styles.settingAdjustment}>
+                    <OptionIcon />
+                </View>
+
+                {/* view created to shift settings icon slightly up */}
+                <View style={{ flex: 1 }} />
+            </TouchableOpacity>
+
+            <View style={styles.imageContainer}>
+                <SistemaLogo />
+            </View>
+            
+            {/* This view created to center image container in 1: 5 : 1 ratio */}
+            <View style={{ flex: 1 }} />
```

Comment:
Instead of doing this, you might be able to use [alignSelf](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-align-self) for both the settings icon (shift it to the end) and logo (align it center).